### PR TITLE
fix: path.sep at win

### PR DESCRIPTION
**Pull request by atian25**
PR to see win test

### CODE REVIEW

**Comment by atian25** [comment]
@popomore win 测试没开?

File: index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function resolveEntry(options) {
 function getFiles(files, cwd) {
   const result = new Set();
   for (let file of files) {
-    file = path.relative(cwd, file).split('/')[0];
+    file = path.relative(cwd, file).split(path.sep)[0];
```
Comment (atian25) [inline index.js:83]: fix win bug

**Comment by atian25** [comment]
image

seems `egg-bin` import `egg-utils@2.1.0` break node 4.x, should we fix `egg-bin` with 4.x support or drop it ?

**Comment by popomore** [comment]
用 egg-bin 1 把

**Comment by atian25** [comment]
这个改为 egg-bin@1 了.

但 egg-bin@2 那边要看怎么搞，它本身是声明支持 4.x 的

**Comment by popomore** [comment]
+ ypkgfiles@1.3.1

**Pull request merged**